windows/RNSVG/Utils.h

```diff
@@ -0,0 +1,341 @@
+    // translate(translate-x, translate-y) scale(scale-x, scale-y).
+    auto const &translate{Numerics::make_float3x2_translation(translateX, translateY)};
+    auto const &scale{Numerics::make_float3x2_scale(scaleX, scaleY)};
+
+    return scale * translate;
+  }
+
+  static RNSVG::MeetOrSlice GetMeetOrSlice(JSValue const &value) {
+    if (value.IsNull()) {
+      return RNSVG::MeetOrSlice::Meet;
+    }
+
+    switch (value.AsInt8()) {
+      case 2:
+        return RNSVG::MeetOrSlice::None;
+      case 1:
+        return RNSVG::MeetOrSlice::Slice;
+      case 0:
+      default:
+        return RNSVG::MeetOrSlice::Meet;
+    }
+  }
+
+  static std::string JSValueAsBrushUnits(JSValue const &value, std::string defaultValue = "objectBoundingBox") {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else {
+      switch (value.AsInt32()) {
+        case 1:
+          return "userSpaceOnUse";
+        case 0:
+        default:
+          return "objectBoundingBox";
+      }
+    }
+  }
+
+  static float JSValueAsFloat(JSValue const &value, float defaultValue = 0.0f) {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else {
+      return value.AsSingle();
+    }
+  }
+
+  static std::string JSValueAsString(JSValue const &value, std::string defaultValue = "") {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else {
+      return value.AsString();
+    }
+  }
+
+  static Color JSValueAsColor(JSValue const &value, Color defaultValue = Colors::Transparent()) {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else if (auto const &brush{value.To<Xaml::Media::Brush>()}) {
+      if (auto const &scb{brush.try_as<Xaml::Media::SolidColorBrush>()}) {
+        return scb.Color();
+      }
+    }
+
+    return defaultValue;
+  }
+
+  static SVGLength JSValueAsSVGLength(JSValue const &value, SVGLength const &defaultValue = {}) {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else {
+      return RNSVG::implementation::SVGLength::From(value);
+    }
+  }
+
+  static Numerics::float3x2 JSValueAsTransform(JSValue const& value, Numerics::float3x2 defaultValue = {}) {
+    if (value.IsNull()) {
+      return defaultValue;
+    } else {
+      auto const &matrix{value.AsArray()};
+
+      return Numerics::float3x2(
+          matrix.at(0).AsSingle(),
+          matrix.at(1).AsSingle(),
+          matrix.at(2).AsSingle(),
+          matrix.at(3).AsSingle(),
+          matrix.at(4).AsSingle(),
+          matrix.at(5).AsSingle());
+    }
+  }
+
+  static std::vector<Brushes::CanvasGradientStop> JSValueAsStops(JSValue const& value) {
+    if (value.IsNull()) {
+      return {};
+    }
+
+    auto const &stops{value.AsArray()};
+    std::vector<Brushes::CanvasGradientStop> canvasStops{};
+
+    for (size_t i = 0; i < stops.size(); ++i) {
+      Brushes::CanvasGradientStop stop{};
+      stop.Position = Utils::JSValueAsFloat(stops.at(i));
+      stop.Color = Utils::JSValueAsColor(stops.at(++i));
```

Comment:
Curious, does this PR mean PlatformColor as colors would be supported on windows for RN-SVG?

Reply:
This implementation is set up to support PlatformColor, but it will not work right now for the same reason it does not currently work in other platforms - the javascript side of the module is doing its own [color value parsing](https://github.com/react-native-svg/react-native-svg/blob/develop/src/lib/extract/extractColor.ts) instead of using react-native's [processColor](https://github.com/facebook/react-native/blob/main/Libraries/StyleSheet/processColor.js).